cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -678,6 +678,22 @@
     </description>
   </property>
 
+  <property>
+    <name>data.local.storage.user.table.max.open.files</name>
+    <value>100</value>
+    <description>
+      Maximum number of open files per user table.
```

Comment:
Maximum number of open file descriptors per user table to be used for local storage.

Reply:
While re-writing this description, the thing that is not clear to me is if this config is per user table and across all user tables?

Reply:
I think the "per user table" makes the distinction clear.